### DISCUSSION TITLE
Mastodon mirror of the Twitter feed?

### DISCUSSION THREAD

**Discussion by egonw** [wikipathways/GPML-Repo#98] [Ideas]
@khanspers, @AlexanderPico, @mkutmon, all, maybe you have heard about the #twitterMigration to [Mastodon](https://joinmastodon.org/). Many scholars have already gone, but it's very much still in progress. I would like to suggest to set up a Mastodon account on the https://fosstodon.org/about and mirror that with Twitter, meaning that all tweets will automatically be posted on the Mastodon network too with https://moa.party/. Would that be acceptable? Then I can boost (=RT) posts on that network (PS, I am https://scholar.social/@egonw).

**Comment by AlexanderPico** [comment]
This is timely for another reason as well:  The bulk of tweets on our WP account are by Kristina who solely maintains the Tumblr blog with WP-related publications. This is no longer sustainable.

Ideally, we could devise a way to automate (or otherwise share the burden for) the collection of WP-related figures, post them to Tumblr, and then _toot_ and _tweet_ them (yes, Mastodon calls their posts "toots").  

It sounds like Egon has figured out an auto-toot system (awesome), but we really should automate the upstream too.

PMC -> Tumblr -> Tweet -> Toot  _(or another order or other steps?)_

Suggestions?

**Comment by Chris-Evelo** [comment]
I would involve Susan in that discussion. She told me she would be more involved in curation activities by working directly with Kristina. Maybe they can think of something together. But automation of part of the work (finding the papers, and actually pushing the publications) is a good idea too

**Comment by khanspers** [comment]
Adding @susancoort.

I agree with all previous comments. The Tumblr -> Tweet connection can be automated, we had it like that before but I thought the tweets looked ugly so I disabled it. But it's functional if the main focus is automation. We use this for the Cytoscape account.

**Comment by khanspers** [comment]
So https://fosstodon.org/about has stopped new registrations for now because their servers are overwhelmed. I will keep checking, but @egonw if you see it open again please go ahead with the registration.

**Comment by khanspers** [comment]
We now have a Mastodon account @wikipathways@fosstodon.org (thanks to Egon for setting it up), and a moa.party link to auto-tweet toots on our WikiPathways Twitter. You may have seen the first auto-tweet already. 
Right now the account is using my email, but we're looking to see if it's possible to have multiple account managers.

**Reply by egonw** [reply]
> Right now the account is using my email, but we're looking to see if it's possible to have multiple account managers.

I got one reply, saying they shared the login credentials.
